src/pudl/transform/eia860.py

```diff
@@ -748,3 +748,51 @@ def clean_boilers_eia860(
     )
 
     return b_df
+
+
+@asset
+def clean_emissions_control_equipment_eia860(
+    raw_emissions_control_equipment_eia860: pd.DataFrame,
+) -> pd.DataFrame:
+    """Pull and transform the emissions control equipment table."""
+    # Replace empty strings, whitespace, and '.' fields with real NA values
+    emce_df = pudl.helpers.fix_eia_na(raw_emissions_control_equipment_eia860)
+
+    # Fix bad months
+    emce_df["operating_month"] = emce_df["operating_month"].replace({"88": "8"})
+    emce_df["retirement_month"] = emce_df["retirement_month"].replace({"True": np.nan})
+
+    # Convert column to boolean
+    emce_df = pudl.helpers.convert_col_to_bool(
+        df=emce_df, col_name="acid_gas_control", true_values=["Y"], false_values=[]
+    )
+    # Add a emission_control_id_pudl as a primary key. This is not unique over years.
+    # We could maybe try and do this, but not doing it now.
+    emce_df["emission_control_id_pudl"] = (
+        emce_df.groupby(["report_year", "plant_id_eia"]).cumcount() + 1
+    )
```

Comment:
Decided not to waste any effort trying to make this value identifiable over time.